lib/types/platform/gitlab/index.ts

```diff
@@ -12,3 +13,10 @@ export type GitlabTreeNode = {
   id: string;
   name: string;
 } & GitTreeNode;
+
+export const GITLAB_API_USING_HOST_TYPES = [
+  PLATFORM_TYPE_GITLAB,
+  'gitlab-releases',
+  'gitlab-tags',
+  'pod',
+];
```

Comment:
Can you add verification tests like in https://github.com/renovatebot/renovate/blob/main/lib/types/github.spec.ts